mail/postfix/src/opnsense/mvc/app/controllers/OPNsense/Postfix/forms/general.xml

```diff
@@ -42,6 +44,24 @@
         <help>The smtpd_banner parameter specifies the text that follows the 220 code in the SMTP server's greeting banner. Default is "'System Hostname' ESMTP Postfix".</help>
     </field>
     <field>
+        <id>general.message_size_limit</id>
+        <label>Message Size Limit</label>
+        <type>text</type>
+        <help>Set the max size for messages to accept, default is 501200000 Byte which is 50MB. Values must be entered in Bytes.</help>
+    </field>
+    <field>
+        <id>general.disable_ssl</id>
+        <label>Disable old SSL</label>
+        <type>checkbox</type>
+        <help>Disable SSLv2 and SSLv3, only TLS allowed.</help>
+    </field>
+    <field>
+        <id>general.disable_weak_ciphers</id>
+        <label>Disable weak Ciphers</label>
+        <type>checkbox</type>
+        <help>This will disable known weak ciphers like DES, RC4 or MD5.</help>
```

Comment:
MD5 is not a cipher. It is a diggest algorithm.

Reply:
sorry, only one g => digest

Reply:
ciphers comes from postfix, so also wrong there ;)

Disable weak ciphers and algorithms?

Reply:
sounds good

Reply:
fixed